Helper/Data.php

```diff
@@ -213,9 +213,9 @@ public function __construct(
     public function getRecurringTypes()
     {
         return [
-            \Adyen\Payment\Model\RecurringType::ONECLICK => 'ONECLICK',
-            \Adyen\Payment\Model\RecurringType::ONECLICK_RECURRING => 'ONECLICK,RECURRING',
-            \Adyen\Payment\Model\RecurringType::RECURRING => 'RECURRING'
+            \Adyen\Payment\Model\RecurringType::ONECLICK => 'One-click',
+            \Adyen\Payment\Model\RecurringType::ONECLICK_RECURRING => 'One-click,Recurring',
+            \Adyen\Payment\Model\RecurringType::RECURRING => 'Recurring'
```

Comment:
This will break the implementation. These are not labels these are send into the request. We need to send the values ONECLICK, ONECLICK,RECURRING or RECURRING.